app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -642,45 +644,59 @@ struct DeviceDelegateOculusVR::State {
       controllerTransform = vrb::Matrix::Rotation(quat);
     }
 
-    if (controllerCapabilities.ControllerCapabilities & ovrControllerCaps_HasPositionTracking) {
-      auto & position = tracking.HeadPose.Pose.Position;
-      controllerTransform.TranslateInPlace(vrb::Vector(position.x, position.y, position.z));
-    } else {
-      controllerTransform = elbow->GetTransform(hand, head, controllerTransform);
-    }
-
+    controllerTransform = elbow->GetTransform(hand, head, controllerTransform);
```

Comment:
6Dof controllers should not require an elbow model.

Reply:
got it. it works after add `kAverageHeight` as we give to the headset position.